README.md

```diff
@@ -15,6 +15,39 @@ for Windows' [Google Group](http://groups.google.com/group/git-for-windows),
 and [contribute bug
 fixes](https://github.com/git-for-windows/git/wiki/How-to-participate).
 
+To build Git for Windows, please either install [Git for Windows'
+SDK](https://gitforwindows.org/#download-sdk), start its `git-bash.exe`, `cd`
+to your Git worktree and run `make`, or open the Git worktree as a folder in
+Visual Studio.
+
+To verify that your build works, use one of the following methods:
+
+- If you want to test the built executables within Git for Windows' SDK,
+  prepend `<worktree>/bin-wrappers` to the `PATH`.
+- Alternatively, run `make install` in the Git worktree.
+- If you need to test this in a full installer, run `sdk build
+  git-and-installer`.
+- You can also "install" Git into an existing portable Git via `make install
+  DESTDIR=<dir>` where `<dir>` refers to the top-level directory of the
+  portable Git.
+- If you built using Visual Studio, you can use the menu item `Build>Install
+  git` (you will want to click on `Project>CMake Settings for Git` first, then
+  click on `Edit JSON` and then point `installRoot` to the `mingw64` directory
+  of an already-unpacked portable Git).
```

Comment:
What version of VS do they need to do this?  Probably VS2019 and maybe even a recent dot release, right?  I don't remember if VS2017 handled this or not.

Reply:
I am unsure about that, so I left this open on purpose. In general, I do not recommend using anything but the newest Visual Studio anyway. I mean, yes, there was a contributor who as recently as two years ago wanted to make things work with Visual Studio 2008, but you know, that'd quite a bit of maintenance burden to support older versions.

Reply:
I wasn't suggesting that we support older versions. No, no.  I was only saying that we just state that they need a recent version of VS.  and if we knew what that version was, we could say it here.

Reply:
Okay, fair enough. How about calling it a "recent Visual Studio" so that I do not have to look up which one is the first to open the folder correctly, and also to avoid having to update this in the future?